src/wms/src/main/java/org/geoserver/wms/map/DirectRasterRenderer.java

```diff
@@ -243,7 +247,11 @@ public RenderedImage render() throws FactoryException {
         try {
             final Color readerBgColor = transparent ? null : bgColor;
             CoordinateReferenceSystem mapCRS = mapEnvelope.getCoordinateReferenceSystem();
-            if (transformation == null && wms.isAdvancedProjectionHandlingEnabled()) {
+            boolean isAdvancedProjectionHandling =
```

Comment:
It's a variable, not a method.

```suggestion
            boolean advancedProjectionHandling =
```

Reply:
Fixed